e2e/run-e2e-tests.sh

```diff
@@ -323,6 +323,7 @@ check_nftables() {
   read -d '' POSTROUTING_RULES_WORKER << EOM
 table ip flannel-ipv4 {
 	chain postrtg {
+    comment "chain to manage traffic masquerading by flannel"
```

Comment:
I think it'd look better if it is aligned with the rest of the related config lines of the chain

Reply:
for some reason vscode put spaces instead of tabs. it breaks the test anyway